gcore/src/msg.rs

```diff
@@ -171,7 +171,7 @@ pub fn read_at(offset: usize, buffer: &mut [u8]) -> Result<()> {
         unsafe {
             gsys::gr_read(
                 offset as u32,
-                size as u32,
```

Comment:
Call of `gcore::size()` must be removed from the beginning of func then as well as check for zero length adapted?

Reply:
size is still used to return error if out of bounds